WordPress/src/main/res/values/strings.xml

```diff
@@ -1920,6 +1922,7 @@
     <string name="login_already_logged_in_wpcom">Already logged in to WordPress.com</string>
     <string name="login_username_at">\@%s</string>
     <string name="enter_site_address">Enter the address of your WordPress site you\'d like to connect.</string>
+    <string name="enter_site_address_share_intent">Enter the address of your WordPress site you\'d like to share the content to</string>
```

Comment:
Let's add a final period `.` to this string to match its sibling (`enter_site_address`). Thanks!

Reply:
Fixed